server.js

```diff
@@ -0,0 +1,163 @@
+const cluster = require('cluster')
+const express = require('express')
+const bodyParser = require('body-parser')
+const app = express()
+const fs = require('fs')
+const r = require('rethinkdbdash')()
+const config = require('./config.json')
+
+const cpusLength = require('os').cpus().length
+app.use(bodyParser.json())
+
+// DBL webhooks
+app.post('/dblwebhook', async (req, res) => {
+  if (req.headers.authorization) {
+    if (req.headers.authorization === config.dbl_webhook_secret) {
+      req.body.type === 'upvote' ? await addPocket(req.body.user, 25)
+        : console.log('not an upvote??')
+      res.send({status: 200})
+    } else {
+      res.send({status: 401, error: 'You done gone goofed up auth.'})
+    }
+  } else {
+    res.send({status: 403, error: 'Pls stop.'})
+  }
+})
+
+// DBL webhooks
+app.post('/patreonwebhook', async (req, res) => {
+    if (req.headers['X-Patreon-Signature']) {
+      if (req.headers['X-Patreon-Signature'] /*=== The HEX digest of the message body HMAC signed (using MD5) using the webhook's secret*/) {
+        if (req.body.data.attributes.patron_status === "active_patron") {
+            await addDonor(res.body);
+        } else if (req.body.data.attributes.patron_status === "former_patron") {
+            await removeDonor();
+        }
+        res.send({status: 200})
+      } else {
+        res.send({status: 401, error: 'You done gone goofed up auth.'})
+      }
+    } else {
+      res.send({status: 403, error: 'Pls stop.'})
+    }
+})
+
+async function addDonor(body) {
+    if (!body.data.included[1].attributes.social_connections.discord || !body.data.included[1].attributes.social_connections.discord.user_id) {
+        return;
+    }
+    return r.table('donors')
+    .insert({
+      id: body.data.included[1].attributes.social_connections.discord.user_id,
+      donorAmount: body.data.attributes.currently_entitled_amount_cents / 100,
+      guilds: [],
+      guildRedeems: 0,
+      firstDonationDate: body.data.attributes.pledge_relationship_start || r.now(),
+      declinedSince: null,
+      totalPaid: donorAmount,
+      patreonID: body.data.included[1].attributes.id,
+    }, { conflict: 'update' })
+    .run()
+}
+
+function removeDonor(body) {
+    return r.table('donors')
+      .getAll(id, {index: 'patreonID'})
+      .delete()
+      .run()
+}
+
+function launchServer () {
+  const http = require('http')
+  http.createServer(app).listen(8200)
+  console.log(`Server started on port 8200 pid: ${process.pid}`)
+}
+
+if (cluster.isMaster) {
+  const workerNumber = cpusLength - 1
+  console.log(`Starting ${workerNumber} workers`)
+  for (let i = 0; i < workerNumber; i++) {
+    cluster.fork()
+  }
+  for (const id in cluster.workers) {
```

Comment:
I don't think you'll need the message listeners for the workers unless you want something from them for some reason. We originally had this to exchange process stats like how much RAM and the amount of times an endpoint was called because that's stored per process and can't share with the master.